dags/analyzer_helper/tests/integration/test_extract_raw_members.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+import unittest
+from analyzer_helper.discourse.extract_raw_members import ExtractRawMembers
+from github.neo4j_storage.neo4j_connection import Neo4jConnection
+
+class TestExtractRawMembers(unittest.TestCase):
+
+    def setUp(cls):
+        cls.neo4jConnection = Neo4jConnection()
+        cls.driver = cls.neo4jConnection.connect_neo4j()
+        cls.test_forum_endpoint = "https://test-forum.discourse.org"
+        cls.platform_id = "test_platform"
+        cls.extractor = ExtractRawMembers(cls.test_forum_endpoint, cls.platform_id)
+        cls.rawmembers_collection = cls.extractor.rawmembers_collection
+        
+        cls.rawmembers_collection.insert_many([
+            {'id': 1, 'joined_at': datetime(2023, 7, 1), 'avatar': 'avatar1.png', 'badgeIds': [1, 2]},
+            {'id': 2, 'joined_at': datetime(2023, 2, 2), 'avatar': 'avatar2.png', 'badgeIds': [3]}
```

Comment:
This data is not representative of the right general data structure. Please update it.